xml/depl_maintenance.xml

```diff
@@ -508,6 +508,262 @@
+       the problems.
+      </para>
+     </step>
+     <step>
+      <para>
+       Stop all &ostack; services that make use of the database to prevent them
+       from writing new data during the migration. Stop shef-client services on
+       the nodes, to prevent regular runs from starting database-related
+       &ostack; services again. <emphasis>From this point, &ostack; services
+       become unavailable.</emphasis>
+      </para>
+     </step>
+     <step>
+      <para>
+       You can now migrate databases using the psql2mysql tool. To upgrade a
+       single database, use the following command format:
+      </para>
+<screen>psql2mysql \
+--source postgresql://neutron:secret@192.168.1.1/neutron \
+--target mysql+pymysql://neutron:evenmoresecret@192.168.1.2/neutron?charset=utf8 \
+migrate</screen>
+      <para>
+       To migrate all databases in one operation, use the
+       <literal>--batch</literal> option and the
+       <filename>/etc/pg2mysql/databases.yaml</filename> file as follows:
+      </para>
+<screen>psql2mysql.py --batch /etc/pg2mysql/databases.yaml migrate</screen>
+     </step>
+     <step>
+      <para>
+       In the &crow; &wi;, switch to the <guimenu>Database</guimenu>
+       &barcl;. Enable the raw view and set the value of
+       <literal>sql_engine</literal> to <literal>mysql</literal>. Apply the
+       &barcl;. After this step, &ostack; services should be running again and
+       reconfigured to use the &mariadb; database back end.
+      </para>
+     </step>
+     <step>
+      <para>
+        To prevent the &postgres;-related chef code from running, unassign the values
+	from <literal>database-server</literal> role in the <guimenu>Database</guimenu>
+       &barcl;, and apply the &barcl;.
+      </para>
+     </step>
+     <step>
+      <para>
+       Start chef-client services on the nodes again.
+      </para>
+     </step>
+     <step>
+      <para>
+       Stop &postgres; on the &dbnode;s. Uninstall &postgres;packages.
+      </para>
+      <substeps>
+       <step>
+	<para>
+	 To stop the <literal>postgresql</literal> service, run the following
+	 command on each cluster node:
+	</para>
+<screen>crm resource stop postgresql
+crm resource stop fs-postgresql
+crm resource stop drbd-postgresql</screen>
+        <para>
+	 Run the last command only if the previous setup used DRBD.
+	</para>
+       </step>
+       <step>
+	<para>
+	 Remove the packages on all cluster nodes:
+	</para>
+	<screen>zypper rm postgresql94 postgresql94-server</screen>
+       </step>
+       <step>
+	<para>
+	 If you choose not to upgrade to &productname; 8 right away, delete unused pacemaker resource from each cluster node:
+	</para>
+<screen>crm conf delete drbd-postgresql
+crm conf delete fs-postgresql
+crm conf delete postgresql</screen>
+       </step>
+      </substeps>
+     </step>
+     <step>
+      <para>
+       While you can continue using DRBD with &rabbit;, it is recommended to
```

Comment:
Ah, we have to change this.
As mentioned earlier, RabbitMQ switch does not work before the upgrade. So we must write something like:

"If DRBD is not used as a backend for RabbitMQ, it is possible to remove it at this point. Delete the packages with zypper and claim the DRBD disks if you want. Otherwise, leave this step until after the full upgrade is done. RabbitMQ setup will be switched from DRBD during the upgrade procedure".